.gitlab-ci.yml

```diff
@@ -8,6 +8,7 @@ stages:
 
 variables:
   UNVERSIONED_REPOSITORY: "true"
+  GIT_DEPTH: 0 # avoid shallow clone, this test requires full git history
```

Comment:
To prevent propagating it to undesired places, it should be set on the `test:unit: ` job